src/sage_docbuild/__main__.py

```diff
@@ -288,6 +288,9 @@ def setup_parser():
     standard.add_argument("--no-plot", dest="no_plot",
                           action="store_true",
                           help="do not include graphics auto-generated using the '.. plot' markup")
+    standard.add_argument("--no-preparsed-examples", dest="no_preparsed_examples",
```

Comment:
What's the use case for this option?

Reply:
It's faster and some people may not want to see the tabs.

Reply:
Speedwise it doesn't seem to make a change (by looking at the github workflow execution time). Thus, to reduce complexity I prefer to just remove the option here and always build the docs with the parsed python tab.

Reply:
Noted that this is your preference. I won't remove it.

Reply:
Asked and answered already.

Reply:
Not really. I don't think we should start to make various features of the documentation configurable (except for the live docs that need additional setup non of the features can be toggled off), it just bloats the code and increases the maintenance effort without any real benefits. So please remove this.

Reply:
You're repeating yourself.

Reply:
More rules for review
- Check the facts about an unfamiliar domain before making claims about it

Specifically here: Any developer who has worked with the Sage docbuild system knows that we have long had a range of options that configure global aspects of it. There's `SAGE_SKIP_PLOT_DIRECTIVE` (which you see even in the diff of the present PR...), `SAGE_DOC_UNDERSCORE`, `SAGE_USE_CDNS`, and we used to have `SAGE_DOC_MATHJAX`, `SAGE_DOC_JSMATH` until recently.

Reply:
In each of these cases there is a good reason for why the switch exists beyond "some people may not want to" have xyz.

Reply:
Just a moment ago you denied that the switches even exist.
Now you say they exist but they have a better reason to exist.

This is called "moving the goalposts", a well-known standard technique.

Reply:
The goalpost stayed the same: we should not add config options to toggle default documentation features off just because "some people may not want to see [xyz]".

Reply:
Yes, you're repeating yourself. 

Just because you have this opinion does not mean that this PR needs work.